### PR TITLE
Change module format to es6

### PR DESCRIPTION
**Pull request by Syps**
- change module format to es6 by adding `"type": "module"` in `package.json`
- update unit tests to use `transformAsync`
- run `npm audit fix` to resolve security vulnerabilities
- bump version to `0.1.0`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { importDeclaration, stringLiteral, isStringLiteral, callExpression } from '@babel/types';
-import { NODE14_BUILTIN_MODULES } from './constants';
```
Comment (timothysu) [inline index.js:2]: is the file extension needed? shouldn't be the case if it's es6 compatible

File: babel.config.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-module.exports = {
```
Comment (timothysu) [inline babel.config.js:1]: 👍

File: test/tests.js [hunk 3]
```diff
@@ -10,92 +10,109 @@ const defaultPluginOptions = {
   }
 };
 
-function transform(code, transformOptions = {}, pluginOptions = defaultPluginOptions) {
-  return babelTransform(code, {
+async function transform(code, transformOptions = {}, pluginOptions = defaultPluginOptions) {
```
Comment (timothysu) [inline test/tests.js:13]: i'm assuming you've tried it where it's used in the frontend alraedy, and it deals with this new promise via async/await as well...or `.then()...catch()`?

**Review by timothysu — APPROVED** [review]
LGTM if it works with the front end!

File: index.js (hunk 1 above)
Comment (Syps) [reply index.js:2]: needed for relative imports https://nodejs.org/api/esm.html#terminology

File: test/tests.js (hunk 3 above)
Comment (Syps) [reply test/tests.js:13]: ya front end worked! for some reason i could only get the tests to run if I used async

**Comment by timothysu** [comment]
🚀 !

**Pull request merged**